### PR TITLE
Add test sock proxy

### PR DESCRIPTION
**Pull request by horia-delicoti**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -39,4 +39,11 @@ RUN chmod a+x entry.sh
 
 ADD lib lib
 
+RUN apt-get update && apt-get install -y unzip
```
Comment (horia-delicoti) [inline Dockerfile]: This line will be moved to [L19](https://github.com/resin-io/resinos-tests/pull/205/files#diff-3254677a7917c6c01f55212f86c57fbfL19)

Comment (jviotti) [reply Dockerfile]: Not sure I understand this comment. Are you moving it there now?

File: lib/main.js [hunk 2]
```diff
@@ -190,7 +221,7 @@ ava.test('Device has started application', async (test) => {
   test.is(commit.length, 40)
 })
 
-ava.test(`Resin host OS update [${options.resinOSVersion} -> ${options.resinOSVersionUpdate}]`, async (test) => {
+ava.test.skip(`Resin host OS update [${options.resinOSVersion} -> ${options.resinOSVersionUpdate}]`, async (test) => {
```
Comment (jviotti) [inline lib/main.js]: Any reason to skip these tests?

File: lib/main.js [hunk 3]
```diff
@@ -33,6 +33,7 @@ const options = {
   email: process.env.RESINOS_TESTS_EMAIL,
   password: process.env.RESINOS_TESTS_PASSWORD,
   tmpdir: process.env.RESINOS_TESTS_TMPDIR,
+  proxySocks:process.env.RESINOS_TESTS_PROXY_SOCKS,
```
Comment (jviotti) [inline lib/main.js]: You're missing a space after the colon here.

File: lib/main.js [hunk 4]
```diff
@@ -126,6 +127,36 @@ ava.test.before(async () => {
...
   })
 
+  if (options.proxySocks) {
+    console.log('Forwarding SOCKS proxy')
+    await resinio.sshHostOS('mkdir -p /mnt/boot/system-proxy/',
+      context.uuid,
+      context.key.privateKeyPath
+    )
+    console.log('this was that')
+    // await resinio.sshHostOS('cat <<\'EOF\'>> /mnt/boot/system-proxy/redsocks.conf\n' +
```
Comment (jviotti) [inline lib/main.js]: Can should probably create the dir above and put this file in one single command, so we don't open and close an SSH connection twice

File: Dockerfile (hunk 1 above)
Comment (horia-delicoti) [reply Dockerfile]: Yes, I will move it but after the test will run correctly. Right now, this is just wip.

File: lib/main.js (hunk 2 above)
Comment (horia-delicoti) [reply lib/main.js]: Same, skipped just for testing.

**Comment by telphan** [comment]
Already implemented.

**Pull request closed without merging**